src/main/background-processes/webdav.ts

```diff
@@ -60,6 +61,8 @@ eventBus.on('USER_LOGGED_IN', () => {
   }
 });
 
+eventBus.on('APP_IS_READY', ejectMacOSInstallerDisks);
```

Comment:
Worth to add a .catch to the promise, so if this fails we:

A) Handle the error correctly
B) Avoid unhandled exceptions in case it fails, exceptions thrown from an eventEmitter listener are hard to debug
C) Can report the error + context to Sentry if we want

Also I think we should register this listener on MacOS only, so we don't have a NOOP listening on Windows + Linux